app-emulation/docker/docker-9999.ebuild

```diff
@@ -223,25 +226,11 @@ src_compile() {
 		fi
 	done
 
-	if use seccomp; then
-		DOCKER_BUILDTAGS+=" seccomp"
-	fi
-	if use selinux; then
-		DOCKER_BUILDTAGS+=" selinux"
-	fi
-	if use apparmor; then
-		DOCKER_BUILDTAGS+=' apparmor'
-	fi
-	if use journald; then
-		DOCKER_BUILDTAGS+=' journald'
-	fi
-
-	if has_version '<sys-fs/lvm2-2.02.110' ; then
```

Comment:
i mistakenly removed it, @glevand added it for arm64 so i think it should be kept.